pkg/minikube/bootstrapper/certs.go

```diff
@@ -76,7 +97,7 @@ func SetupCerts(cmd command.Runner, k8s config.KubernetesConfig) error {
 	}
 
 	kubeCfg := api.NewConfig()
-	err := util.PopulateKubeConfig(kubeCfgSetup, kubeCfg)
+	err = util.PopulateKubeConfig(kubeCfgSetup, kubeCfg)
```

Comment:
this was renamed to  kubeconfig.PopulateFromSettings

https://github.com/kubernetes/minikube/blob/master/pkg/minikube/kubeconfig/settings.go#L67